src/components/product-switcher/index.tsx

```diff
@@ -53,6 +55,9 @@ const ProductSwitcher: React.FC = () => {
     })
   )
 
+  /* @TODO: handle case where there is no
```

Comment:
Have taken a VERY rough shot at this, but it doesn't feel like a proper fix (just one oriented at getting the demo done), so have added this `TODO`.